docs/pbc/all/tax-management/202307.0/vertex/integrate-vertex.md

```diff
@@ -0,0 +1,520 @@
+
+Add the config to `src/Pyz/Zed/Oms/OmsDependencyProvider.php`:
+
+```php
+// ...
+    
+use Spryker\Zed\TaxApp\Communication\Plugin\Oms\Command\SendPaymentTaxInvoicePlugin;
+
+// ...
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function extendCommandPlugins(Container $container): Container
+    {
+         $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
+             
+             // ...
+             
+             $commandCollection->add(new SendPaymentTaxInvoicePlugin(), 'TaxApp/SubmitPaymentTaxInvoice');
+
+             // ...
+            
+             return $commandCollection;
+        });
+        
+        return $container;
+    }
+
+```
+
+### 4. Configure Calculation Dependency Provider
+
+Add the following to `src/Pyz/Zed/Calculation/CalculationDependencyProvider.php`
+
+```php
+// ...
+
+use Spryker\Zed\TaxApp\Communication\Plugin\Calculation\TaxAppCalculationPlugin;
+
+// ...
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CalculationExtension\Dependency\Plugin\CalculationPluginInterface>
+     */
+    protected function getQuoteCalculatorPluginStack(Container $container): array
+    {
+        /** @var array<\Spryker\Zed\Calculation\Dependency\Plugin\CalculationPluginInterface> $pluginStack */
+        $pluginStack = [
+            // ...
+        
+            new TaxAppCalculationPlugin(),
+        
+            // ...
+        ];
+        
+        return $pluginStack;
+    }
+
+// ...
+```
+
+### 5. Configure Tax App Dependency Provider
+
+Tax App Dependency Provider could have next configuration `src/Pyz/Zed/TaxApp/TaxAppDependencyProvider.php`:
+
+```php
+<?php
+
+namespace Pyz\Zed\TaxApp;
+
+// ...
+
+class TaxAppDependencyProvider extends SprykerTaxAppDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\TaxAppExtension\Dependency\Plugin\CalculableObjectTaxAppExpanderPluginInterface>
+     */
+    protected function getCalculableObjectExpanderPluginCollection(): array
+    {
+        return [
+            new CalculableObjectCustomerWithTaxCodeExpanderPlugin(),
+            new CalculableObjectExpensesWithTaxCodeExpanderPlugin(),
+            new CalculableObjectItemProductOptionWithTaxCodeExpanderPlugin(),
+            new CalculableObjectItemWithTaxTaxCodeExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\TaxAppExtension\Dependency\Plugin\OrderTaxAppExpanderPluginInterface>
+     */
+    protected function getOrderExpanderPluginCollection(): array
+    {
+        return [
+            new OrderCustomerWithTaxCodeExpanderPlugin(),
+            new OrderExpensesWithTaxCodeExpanderPlugin(),
+            new OrderItemProductOptionWithTaxCodeExpanderPlugin(),
+            new OrderItemWithTaxTaxCodeExpanderPlugin(),
+        ];
```

Comment:
where to get these plugins? please add a namespace to the `use` section

Reply:
OK, I see that it's defined in the info box section

Reply:
additionally this codebox is duplicated at the end ("So in general the plugin stack could look like:"). So maybe it makes sense to remove it from here.